microservices-dashboard-server/src/main/java/be/ordina/msdashboard/aggregator/health/HealthAggregator.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package be.ordina.msdashboard.aggregator.health;
+
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import be.ordina.msdashboard.LandscapeWatcher;
+import be.ordina.msdashboard.events.HealthInfoFailed;
+import be.ordina.msdashboard.events.HealthInfoRetrieved;
+import be.ordina.msdashboard.events.NewServiceInstanceDiscovered;
+
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.Status;
+import org.springframework.cloud.client.ServiceInstance;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.context.event.EventListener;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.util.UriComponentsBuilder;
+
+/**
+ * Aggregator for the /health endpoint of a regular Spring Boot application.
+ *
+ * @author Dieter Hubau
+ */
+@Component("msDashboardHealthAggregator")
```

Comment:
Can you remove this annotation and add the bean creation to `MicroservicesDashboardServerAutoConfiguration`?